pkgs/servers/mattermost/default.nix

```diff
@@ -21,10 +21,12 @@ buildGoPackage rec {
   buildPhase = ''
     runHook preBuild
     cd go/src/${goPackagePath}/cmd/platform
-    go install
+    go install ${buildFlags}
     runHook postBuild
   '';
 
+  buildFlags = "-ldflags \"-X '${goPackagePath}/model.BuildNumber=nixpkgs-${version}'\"";
```

Comment:
Could you please move `buildFlags` up into a `let` expression? This is not needed in the build environment. :)